android/settings.gradle

```diff
@@ -49,3 +51,6 @@ include ':react-native-sqlite-storage'
 project(':react-native-sqlite-storage').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-sqlite-storage/src/android')
 
 include ':app'
+
+include ':react-native-languages'
+project(':react-native-languages').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-languages/android')
```

Comment:
This is a duplicate; react-native-languages is already included further up. Also, the i18n package is misisng here I think (not linked properly).